ci-operator/step-registry/openshift/microshift/infra/wait-for-cluster-up-aws/openshift-microshift-infra-wait-for-cluster-up-aws-commands.sh

```diff
@@ -60,25 +61,13 @@ set +e
 set -x
 PS4='+ $(date "+%T.%N")\011'
 
-retries=3
+retries=10
 while [ ${retries} -gt 0 ] ; do
   ((retries-=1))
-
-  KUBECONFIG="${SHARED_DIR}/kubeconfig" oc wait \
-    pod \
-    --for=condition=ready \
-    -l='app.kubernetes.io/name=topolvm-csi-driver' \
-    -n openshift-storage \
-    --timeout=5m
-  [ $? -eq 0 ] && exit 0
-
-  # Image pull operation sometimes get stuck for topolvm images
-  # Delete topolvm pods to retry image pull operation
-  KUBECONFIG="${SHARED_DIR}/kubeconfig" oc delete \
-    pod \
-    -l='app.kubernetes.io/name=topolvm-csi-driver' \
-    -n openshift-storage \
-    --timeout=30s
+  if ssh "${INSTANCE_PREFIX}" "sudo systemctl status greenboot-healthcheck | grep 'active (exited)'"; then
+    exit 0
+  fi
+  sleep 30
```

Comment:
Let's add a message here about waiting and number of remaining retries?

Reply:
Done